specification/datadog/resource-manager/readme.go.md

```diff
@@ -21,9 +21,38 @@ azure-arm: true
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2023-01
+  - tag: packag-2022-08
+  - tag: package-2022-06
   - tag: package-2021-03
   - tag: package-2020-02-preview
 ```
+### Tag: package-2023-01 and go
+
+These settings apply only when `--tag=package-2023-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2023-01' && $(go)
+output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2023-01-01/$(namespace)
+```
+
+### Tag: package-2022-08 and go
+
+These settings apply only when `--tag=package-2022-08 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-08' && $(go)
+output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2022-08-01/$(namespace)
+```
+
+### Tag: package-2022-06 and go
+
+These settings apply only when `--tag=package-2022-06 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-06' && $(go)
+output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2022-06-01/$(namespace)
+```
```

Comment:
these parts serve a deprecated version of go SDK, we no longer need these. Please revert this.

Reply:
Should i remove the entire file or just revert my changes?